source/documentation/breaking-changes/function-units.liquid

```diff
@@ -0,0 +1,213 @@
+in two phases:
+
+### Phase 1
+
+{% compatibility '1.32.0', false, null, false %}{% endcompatibility %}
+
+Currently, Sass just emits a deprecation warning if you pass a number with no
+unit or a unit other than `%` as a lightness or saturation to any function.
+
+### Phase 2
+
+{% compatibility false, false, null, false %}{% endcompatibility %}
+
+In Dart Sass 2.0.0 color functions will throw errors if they're passed a
+saturation or lightness parameter with no unit or a non-`%` unit.
+
+## Alpha
+
+When specifying a color's alpha value, CSS (as of [Colors Level 4]) allows
+either unitless values between 0 and 1 or `%` values between `0%` and `100%`. In
+most cases Sass follows this behavior, but the functions `color.adjust()` and
+`color.change()` have historically allowed *any* unit, and interpreted it as
+unitless. You could even write `color.change(red, $alpha: 1%)` and Sass would
+return the opaque color `black`.
+
+[Colors Level 4]: https://www.w3.org/TR/css-color-4/#typedef-alpha-value
+
+To fix this issue and bring Sass in line with the CSS spec, we're making changes
+in three phases:
+
+### Phase 1
+
+{% compatibility '1.56.0', false, null, false %}{% endcompatibility %}
+
+Currently, Sass just emits a deprecation warning if you pass a number with any
+unit, including `%`, as an alpha value to `color.change()` or `color.adjust()`.
+
+### Phase 2
+
+{% compatibility false, false, null, false %}{% endcompatibility %}
+
+Next, we'll change the way `%` units are handled for the alpha argument to
+`color.change()` and `color.adjust()`. Alphas with unit `%` will be divided by
+`100%`, converting them to unitless numbers between 0 and 1.
+
+Because this change is a bug fix that improves consistency with other Sass
+functions, it will be made with only a minor version bump. It will be changed at
+minimum three months after Phase 1 is released, to give users time to adjust
+their code and avoid the bug.
+
+[Dart Sass compatibility policy]: https://github.com/sass/dart-sass#compatibility-policy
+
+### Phase 3
+
+{% compatibility false, false, null, false %}{% endcompatibility %}
+
+Finally, in Dart Sass 2.0.0 `color.change()` and `color.adjust()` will throw
+errors if they're passed an alpha parameter with a non-`%` unit. Unitless alphas
+will still be allowed.
+
+## `math.random()`
+
+[The `math.random()` function] has historically ignored units in `$limit` and
+returned a unitless value. For example `math.random(100px)` would drop "px" and
+return a value like `42`.
+
+A future version of Sass will stop ignoring units for the `$limit` argument and
+return a random integer with the same units.
+
+[The `math.random()` function]: /documentation/modules/math#random
+{% endmarkdown %}
+
+{% codeExample 1, false %}
+// Future Sass, doesn't work yet!
+@debug math.random(100px); // 42px
+===
+// Future Sass, doesn't work yet!
+@debug math.random(100px)  // 42px
+{% endcodeExample %}
+
+{% markdown %}
+### Phase 1
```

Comment:
There's a bunch of repeated anchor links because all sections have "Phase 1", "Phase 2", and "Phase 3" headers. In the old site, they all were slugified naively as `#phase-{1,2,3}`, so they actually only point to the first occurrence. The new site is a bit smarter and actually adds extra indeces on repeat headers, so we get `#phase-1-1`, `#phase-1-2`, etc; BUT the indexing only works on single markdown blocks. Because we open and close the markdown block around the `codeExample` tag, we lose the indexing in this `math.random()` section. I think we will just need to add manual `id`s to these headers, or somehow modify the `markdown` tag to accept an offset for slugifying. Thoughts?

Reply:
Converting the whole file to `md` fixed this 🥳